### PR TITLE
Update toolchain to stable

### PR DESCRIPTION
**Pull request by keyboardsurfer**
AGP 7.3 launched. This makes the entire baseline profile toolchain stable.

### CODE REVIEW

File: JankStatsSample/app/build.gradle [hunk 1]
```diff
@@ -41,15 +41,14 @@ android {
 
 dependencies {
     implementation "androidx.appcompat:appcompat:$appcompat_version"
-    implementation "androidx.core:core-ktx:$core_version"
+    implementation "androidx.compose.material:material:$compose_version"
```
Comment (mlykotom) [inline JankStatsSample/app/build.gradle:44]: We had to have 2 versions in the Macrobench sample. Should we have them here as well?

**Review by mlykotom — APPROVED** [review]
One question about compose versioning - we should probably have composeUiVersion and composeToolingVersion?

**Comment by mlykotom** [comment]
The material dependency doesn't exist
https://github.com/android/performance-samples/actions/runs/3067863912/jobs/4954629402#step:4:61

File: JankStatsSample/app/build.gradle (hunk 1 above)
Comment (keyboardsurfer) [reply JankStatsSample/app/build.gradle:44]: It works without, so I'd just keep it this way.

**Comment by keyboardsurfer** [comment]
Fixed dependency issue.

**Pull request merged**